.jenkins/tests/pom.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
While I like the idea of having a PR check for jenkinsfiles, I have concerns about maintainability of this solution:
1/ it brings a lot of dependencies to the project (although it's a separate maven build)
2/ it duplicates the same (or very similar) setup in kogito-runtimes, kogito-examples and kogito-pipeline (in form of build.gradle) and maybe other repositories.

We should think about reducing the maintainability cost before merging the PRs.

First proposal: creating a parent pom.xml (in kogito-pipelines or jenkins-pipeline-shared-libraries ?) the other repositories would inherit from.

Reply:
I see your concerns. I will remove the tests and gonna implement it as part of another issue (I need those PRs to be merged asap)
and I note your proposal for later implementation :)

Reply:
https://issues.redhat.com/browse/BXMSPROD-1125

Reply:
Perfect, thank you!